src/context/modal-context.tsx

```diff
@@ -1,21 +1,27 @@
 import * as React from 'react'
 
-type Modal = {
-  handleOpen(imdb: string): void
-}
+export const ModalContext = React.createContext<any>({})
+
+export function Modal(props: any) {
```

Comment:
👀 👀 👀 👀 👀 👀 👀 👀 `any` 👀 👀 👀 👀 👀 👀 👀 👀 


Hmmmmm, this can be solved with generics, but I am not that experienced with it. 😂